r/vignettes/install.Rmd

```diff
@@ -172,20 +228,13 @@ If downloading dependencies at build time is not an option, as when building on
 
 #### Dependencies for S3 and GCS support
 
-The arrow package allows you to work with data in AWS S3 or in other cloud
-storage system that emulate S3, as well as Google Cloud Storage.
-However, support for working with S3 and GCS is not
-enabled in the default source build, and it has additional system requirements. To
+Support for working with data in S3 and GCS is not enabled in the default
```

Comment:
I think this could be read to conflict with L237 ?

Reply:
Good catch, thanks, I've revised.